f/radio/fn_radioAddHandlers.sqf

```diff
@@ -0,0 +1,101 @@
+// F3 - Folk ARPS Long-Range Radio Module
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+
+/* ========================
+This function adds local event handlers to the player which grant radio channels when picking up a backpack or entering a vehicle. It also makes an initial check to see what they've got to begin with.
+It's activated by f\radio\fn_radioChannels.sqf.
+=========================== */
+
+// Wait for player to be properly initialised
+waitUntil {(!isNull player && {player == player}) && !(isNil "f_var_radioChannelsUnified")};
+
+// Add player to the correct channels if they have a backpack
+[player] call f_fnc_radioCheckChannels;
+
+// Now bail if they've already been handled
+if (player getVariable ["f_var_radioHandlersAdded",false]) exitWith {};
+
+// Update channels if they drop a backpack
+player addEventHandler ["put", { 
+	params ["_unit", "_container", "_item"];
+	[_unit] call f_fnc_radioCheckChannels;
```

Comment:
Since, as far as I can see, whenever we call `f_fnc_radioCheckChannels` we call it twice with a delay to make sure it worked, won't it be better to just add a 2 iteration loop to `f_fnc_radioCheckChannels` itself?

Reply:
Maybe, probably? I'll think about this and come back to it in the morning.

Reply:
@NikkoJT What did you decide here? Personally think it'd be cleaner but happy to approve if you disagree

Reply:
Loop added

Reply:
Awesome, thanks!